lib/takeaway.rb

```diff
@@ -0,0 +1,43 @@
+require 'twilio-ruby'
+
+account_sid = 'sid'
+auth_token = 'token'
+@client = Twilio::REST::Client.new api_key_sid, api_key_secret, account_sid
+
+class Takeaway
+
+  attr_reader :MENU, :order
+  MENU = { "brocolli" => 3, "risotto" => 10, "bread" => 5 }
+
+  def initialize
+    @order = []
+    @menu = MENU
+    @order_total = 0
+  end
+
+  def read_menu
+    MENU
+  end
+
+  def select(pick)
```

Comment:
I suspect that you're creating a nested array here because `select` returns an array?  Try using `find` instead :)